Project.toml

```diff
@@ -4,7 +4,7 @@ keywords = ["testing", "mocking"]
 license = "MIT"
 desc = "Allows Julia function calls to be temporarily overloaded for purpose of testing"
 author = ["Curtis Vogt"]
-version = "0.7.2"
+version = "0.8.0"
```

Comment:
Is this really a breaking change? It seems like we're mostly just fixing a bug.

Reply:
My thought was that it changes behaviour for anyone currently nesting them, hence it could break user code. But if we're happy to define the old behaviour to be a bug, all the better - will change :)